pricelist_by_category_qty/__init__.py

```diff
@@ -0,0 +1,22 @@
+#################################################################################
```

Comment:
Please use limited license format for all your files like : https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L1